configuration-vscode.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <!--
+  Gregorio web site.
+  Copyright (C) 2007-2015 The Gregorio project
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+  <title>Gregorio project website</title>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  <meta name="description" content="Free software for typesetting Gregorian chant" lang="en">
+  <meta name="keywords" content="Gregorian chant, gregorio, gabc, gregoriotex" lang="en">
+  <meta name="author" content="The Gregorio project">
+  <link rel="shortcut icon" href="./illus/fav.ico">
+  <link rel="icon" href="./illus/fav.ico">
+  <link rel="stylesheet" href="./style.css" type="text/css" title="default">
+  <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script type="text/javascript" src="./gregorio-menu.js"></script>
+  <script>
+    var context = 'gregorio';
+    var title = 'Gregorio project website'
+  </script>
+</head>
+
+<body>
+  <div id="ban">
+    <img src="./illus/harpedroite.png" title="" class="hd" alt="" />
+    <a id="haut" name="haut"></a>
+    <div id="banniere">
+    </div>
+  </div>
+  <script>write_gre_menu();</script>
+ <div class="flux">
+    &nbsp;
+    <!-- IE patch-->
+    <div class="text">
+      <!-- End of header -->
+<!-- header -->
+
+<h1>Configuring VSCode for Gregorio</h1>
+
+<h3>Automatic score compilation</h3>
+
+<p>A feature of Gregorio version 3.0 and later is the automatic compilation of gabc scores when needed to complete a <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> document.  For this feature to work, <code>lualatex</code> must be able to launch the <code>gregorio</code> program during compilation.</p>
+
+<p>VSCode can be configured (using the <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>-workshop extension) to do this by creating a new tool and a new recipie for compilation.</p>
+
+<p>To create it, open up the JSON user preferences and add the following under "latex-workshop.latex.tools"</p>
+
+<code style="display: block;white-space: pre-wrap;">
+    {
+        "name": "lualatex",
+        "command": "lualatex.exe",
+        "args": [

```

Comment:
If I understand correctly, this will work only with the Gregorio that comes packaged with TeXLive.  Gregorio updates installed outside the TeXLive system requires the `--shell-escape` option to be active, which I don't see in this configuration.

Now, since we haven't had such an update in several years now, it may not be worth mentioning here.

Reply:
I was getting a warning for providing the `--shell-escape` option which is why I didn't provide it here. I can add a note about this being specifically for the TeXLive version